draft-bgk-ccamp-merged-wdm-tunnel.md

```diff
@@ -177,18 +172,16 @@ contributor:
 
    -  other optical attributes
 
-   Each path can be a tunnel (only defined by source and
-   destination node) or a network tunnel (additionally needs
-   source and destination transponders). Therefore, all the attributes
-   are optional to support both situations.
+Each path can be a tunnel (only defined by the source and destination node)
+or a network tunnel (additionally needs source and destination transponders).
```

Comment:
I think that the terminology in draft-ietf-teas-yang-te distinguishes between end-to-end and segment:

```suggestion
Each path can be a segment path (only defined by the source and destination nodes or link termination points)
or an end-to-end path (additionally needs source and destination transponders).
```